src/Database/Schema/Column/Date.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Part of Windwalker project. 
+ *
+ * @copyright  Copyright (C) 2014 - 2015 LYRASOFT. All rights reserved.
+ * @license    GNU Lesser General Public License version 3 or later.
+ */
+
+namespace Windwalker\Database\Schema\Column;
+
+use Windwalker\Database\Schema\Column;
+use Windwalker\Database\Schema\DataType;
+
+/**
+ * The DATE class.
+ * 
+ * @since  2.0
```

Comment:
Please use 3.2.8 or `__DEPLOY_VERSION__`, build script will replace it.

Reply:
update to 3.2.8